apps/test-suite/tests/NewNotifications.js

```diff
@@ -943,9 +943,61 @@ export async function test(t) {
           },
           16000
         );
+
+        t.it(
```

Comment:
Re:

> Added two integration tests but those tests only verify first day event fire. We may want to remove these are they both take over 1 minute to run. If there are better solutions to verify the notification was scheduled properly, please let me know in the comments.

I think we get away by:
- writing an automatic test that uses `getScheduledNotificationsAsync` to verify that the notification has been scheduled properly with the expected `hour` and `minute` params
- making these **real** tests (scheduling and waiting for the trigger) optional (triggerable with a user interaction, like [here](https://github.com/expo/expo/blob/385e7e0c54594aaf826b3266ea2ecc78a1948d5a/apps/test-suite/tests/NewNotifications.js#L1142-L1145))

Reply:
Sounds good.